print/ghostscript/files/pdf_sec.c.diff

```diff
@@ -0,0 +1,13 @@
+--- pdf/pdf_sec.c.orig	2024-03-06 10:52:27
```

Comment:
A link to the commit that you got the patch from, and a link to the bug it fixes, if available, should go at the top of the patchfile. I also like to include the error message that was encountered.

```suggestion
Fix:

error: assignment to ‘char **’ from incompatible pointer type ‘char *’ [-Wincompatible-pointer-types]
error: assignment to ‘int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]

https://bugs.ghostscript.com/show_bug.cgi?id=707649
https://git.ghostscript.com/?p=ghostpdl.git;a=commitdiff;h=90cabe08422afdd16bac5dd9217602679d943045
--- pdf/pdf_sec.c.orig	2024-03-06 10:52:27
```